LoRaEngine/LoraKeysManagerFacade/IDevice.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
```

Comment:
Enable nullable context in new source files:

```suggestion
#nullable enable

namespace LoraKeysManagerFacade
```

Reply:
@Mandur you should think of enabling for the all repository. I wanted to make the comment as well but was not courageous enough to propose just for a file ;-) You'll have "some" work to enable it everywhere but quite some benefits as well. It's a great way to catch non checked nulls thru the static code analyse made by Roslyn.